app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ActiveRecord::Base
   include AndelaValidator
   include ActionView::Helpers::DateHelper
+  include NewNotification
+
```

Comment:
In another thought, I think the hook should be `after_create` rather than `after_save` so that notifications are not triggered everytime user updates or does some other stuff on the question